keps/sig-auth/2579-psp-replacement/README.md

```diff
@@ -0,0 +1,1085 @@
+required in order to be in the release.
+
+In some cases, the questions below should also have answers in `kep.yaml`. This
+is to enable automation to verify the presence of the review, and to reduce review
+burden and latency.
+
+The KEP must have a approver from the
+[`prod-readiness-approvers`](http://git.k8s.io/enhancements/OWNERS_ALIASES)
+team. Please reach out on the
+[#prod-readiness](https://kubernetes.slack.com/archives/CPNHUMN74) channel if
+you need any help or guidance.
+
+-->
+
+### Feature Enablement and Rollback
+
+_This section must be completed when targeting alpha to a release._
+
+* **How can this feature be enabled / disabled in a live cluster?**
+  - [x] Feature gate (also fill in values in `kep.yaml`)
+    - Feature gate name: `PodSecurity`
+    - Components depending on the feature gate: PodSecurity admission plugin
+  - [x] Other
+    - Describe the mechanism:
+        - The new functionality is entirely encapsulated by the admission controller, so enabling or
+          disabling the feature is a matter of adding or removing the admission plugin from the list
+          of enabled admission plugins.
+    - Will enabling / disabling the feature require downtime of the control
+      plane?
+        - Yes. Admission plugins are statically configured, so enabling or disabling will require an
+          API server restart. This can be done in a rolling manner for HA clusters.
+    - Will enabling / disabling the feature require downtime or reprovisioning
+      of a node? (Do not assume `Dynamic Kubelet Config` feature is enabled).
+        - No. This feature does not touch any node components.
+
+* **Does enabling the feature change any default behavior?**
+  Any change of default behavior may be surprising to users or break existing
+  automations, so be extremely careful here.
+    - No.
+
+* **Can the feature be disabled once it has been enabled (i.e. can we roll back
+  the enablement)?**
+  Also set `disable-supported` to `true` or `false` in `kep.yaml`.
+  Describe the consequences on existing workloads (e.g., if this is a runtime
+  feature, can it break the existing applications?).
+  - Yes. Disabling it means that the policies will no longer be enforced, but there are no stateful
+    changes that will be affected.
+
+* **What happens if we reenable the feature if it was previously rolled back?**
+  - There might be pods violating the policy in namespaces when it's turned on. The [Updates
+    section](#updates) explains how this case is handled.
+
+* **Are there any tests for feature enablement/disablement?**
+  The e2e framework does not currently support enabling or disabling feature
+  gates. However, unit tests in each component dealing with managing data, created
+  with and without the feature, are necessary. At the very least, think about
+  conversion tests if API types are being modified.
+  - There will be tests for updates to "violating" pods, but I do not think explicit feature
```

Comment:
I'm ok with this, but only because of the explicit nature of the admission plugin flags that is tested independent of this feature.